blocks/init/src/Blocks/components/accordion/accordion-style.scss

```diff
@@ -1,12 +1,12 @@
 $accordion: (
 	border-color: global-settings(colors, light),
-	icon-size: 30px,
+	icon-size: 1.5em,
 );
 
 .accordion {
 	$this: &;
 	position: relative;
-	border-bottom: 1px solid map-get-strict($accordion, border-color);
+	border-bottom: 0.05em solid map-get-strict($accordion, border-color);
```

Comment:
We should consider using `rem` instead of `em` in places like these, we wouldn't want the border thickness to vary if a div that the accordion is in has a bigger base font size.